server/datasources.local.js

```diff
@@ -0,0 +1,22 @@
+var debug = require('debug')('MeshModels.server.datasource');
+var url = require('url');
+
+var dbUrl = url.parse(process.env.STRONGLOOP_MESH_DB || 'memory://');
```

Comment:
We might switch to a SL Db instead of Redis/Memory but will probably never back it with a web service.